tools/mixxxdb_cleanup.sql

```diff
@@ -79,6 +79,13 @@ DELETE FROM track_analysis WHERE track_id NOT IN (SELECT id FROM track_locations
 -- Post-cleanup maintenance                                          --
 -----------------------------------------------------------------------
 
+-- Rebuild the entire database file
+-- https://www.sqlite.org/lang_vacuum.html
 VACUUM;
 
-PRAGMA optimize;
```

Comment:
This was removed because we cannot influence when optimization happens, right? I.e. when a user drags a new track onto a deck during a gig we don't want to suddently reanalyze the DB and potentially slow down the computer, correct?

Reply:
Please refer to the SQLite documentation. OPTIMIZE is lazy and only optimizes what the preceding query statistics of the current(!) connection suggest. ANALYZE does not depend on any queries and implies OPTIMIZE.

I am not repeating this here.